tests/Decoders.fs

```diff
@@ -1042,6 +1042,14 @@ Expecting an array but instead got: 1
 
                 equal expected actual
 
+            testCase "keys works" <| fun _ ->
```

Comment:
Can you please add a failing test for `keys` decoder?

Like calling `keys` decoder against a string.

Reply:
Done